tests/Unit/ParallelAlgorithms/LinearSolver/ConjugateGradient/Test_ConjugateGradientAlgorithm.cpp

```diff
@@ -19,9 +19,15 @@ namespace helpers = LinearSolverAlgorithmTestHelpers;
 
 namespace {
 
+struct SerialCG {
```

Comment:
[nitpick] `SerialCG` -> `SerialCg` would be more consistent with most uses of CamelCase on acronyms.